db/schema.rb

```diff
@@ -11,18 +11,18 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20141229224941) do
+ActiveRecord::Schema.define(version: 20150306182232) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
   create_table "active_admin_comments", force: :cascade do |t|
-    t.string   "namespace",     limit: 255
```

Comment:
I have no idea why the schema got string limits after the 4.2 upgrade, or why they've been taken out by resetting the db. The only relevant changes here are on old L148-149 and new L156-157.